hr_attendance_modification_tracking/models/hr_attendance.py

```diff
@@ -11,3 +14,46 @@ class HrAttendance(models.Model):
     employee_id = fields.Many2one(track_visibility="onchange")
     check_in = fields.Datetime(track_visibility="onchange")
     check_out = fields.Datetime(track_visibility="onchange")
+    time_changed_manually = fields.Boolean(
+        string="Time changed",
+        compute="_compute_time_changed_manually",
+        default=False,
+        store=True,
+        help="This attendance has been manually changed by user. If attendance"
+        " is created from form view, a 60 seconds tolerance will "
+        "be applied.",
+    )
+
+    @api.depends("message_ids.tracking_value_ids")
+    def _compute_time_changed_manually(self):
+        for record in self:
+            if not record.time_changed_manually:
+                # For manual attendance, tolerance to consider it acceptable
+                tolerance = timedelta(seconds=60)
+                for track in record.message_ids.mapped("tracking_value_ids"):
```

Comment:
We found that this change has a wrong behavior because of permissions. tracking values should be read with `sudo` / `with_user`. We have added a PR for 12.0 and a commit here.